README.rst

```diff
@@ -1,23 +1,150 @@
-kinto2xml
----------
+#########
+amo2kinto
+#########
 
-Generate a blocklist.xml file from Kinto collections.
+
+Installation
+============
+
+To install the release, just use::
+
+    pip install amo2kinto
+
+
+To install the development environment, just run::
+
+    make install
+
+
+System dependencies
+-------------------
+
+In order to build ``amo2kinto`` dependencies you may need the following libraries:
+
+- Building **lxml** will require: ``libxml2-dev libxslt-dev python-dev``
+- Building **cryptography** will require: ``build-essential libffi-dev libssl-dev python-dev``
+
+If you have got error messages telling you a header file is missing, please install them:
+
+- ``libxml/xmlversion.h: File not found``
+- ``ffi.h: File not found``
+- ``ssl.h: File not found``
+
+
+Importing blocklists from AMO into Kinto
+========================================
+
+The `addons-server <https://github.com/mozilla/addons-server/>`_ is
+able to export its blocklists database in JSON.
+
+You can use the ``json2kinto`` script to load this database into a
+kinto server::
+
+    json2kinto --server http://localhost:8888/v1 \
+               --amo-server https://addons.mozilla.org/  \
+               --schema-file schemas.json
+
+The last version of the schema file can be find here:
+https://github.com/mozilla-services/amo-blocklist-ui/blob/master/amo-blocklist.json
+
+The script is able to synchronize (add new blocked items and remove old ones).
+
+However if a blocked item already exists it won't be altered.
+
+**json2kinto** gives you the ability to configure what you want to
+load, with which user credentials and in which bucket and collection.
 
 .. code-block::
 
-    usage: kinto2xml [-h] [-s SERVER] [-a AUTH] [-b BUCKET] [-v] [-q] [-D]
-                     [-o OUT]
-
-    Build a blocklists.xml file.
-
-    optional arguments:
-      -h, --help            show this help message and exit
-      -s SERVER, --server SERVER
-                            The location of the remote server (with prefix)
-      -a AUTH, --auth AUTH  BasicAuth token:my-secret
-      -b BUCKET, --bucket BUCKET
-                            Bucket name.
-      -v, --verbose         Show all messages.
-      -q, --quiet           Show only critical errors.
-      -D, --debug           Show all messages, including debug messages.
-      -o OUT, --out OUT     Output file.
```

Comment:
Actually I'd prefer having the complete output of `--help`. It's easier to maintain and gives the whole command pattern (`[-s SERVER] [-a AUTH] [-b BUCKET] `)